### DISCUSSION TITLE
Is it possible to reinstall Floorp 11.16.0 again, or any pre-upgrade version

### DISCUSSION THREAD

**Discussion by Telmesomething** [Floorp-Projects/Floorp#1415] [Q&A] [answered]
I've had Floorp installed on Windows 8.1 for some time now where it was running perfectly even though I'm aware it's not supported. However, I inadvertently clicked the upgrade prompt to download 11.18.0 this afternoon and of course that wouldn't install. So now I'm trying to revert to 11.16.0 again  by running the installer for that version, but the only option presented is to upgrade. I've already tried running Windows System Restore which completed successfully, but when I try to run Floorp 11.16.0 the error in the screenshot appears. 

![xpcom error](https://github.com/user-attachments/assets/68d6a2a4-44d7-4fe8-a303-43b0f9517264)

So my question is, is it possible to reinstall the older version again without the error occuring.

**Comment by CutterKnife** [comment]
You can probably install it from the GitHub Releases.
https://github.com/Floorp-Projects/Floorp/releases/tag/v11.16.0

**Reply by Telmesomething** [reply]
I've already tried that using the 11.16.0 installer, but the only option is to upgrade. Here's a pix of it.

![upgrade only](https://github.com/user-attachments/assets/bb22c7f9-78ef-42cc-8c68-1e76f570f45c)

Same thing happens with earlier installers unfortunately.

**Reply by CutterKnife** [reply, marked as answer]
If you uninstall the old Floorp, do you get the same result?
Alternatively, you can try the [Portable version](https://github.com/Floorp-Projects/Floorp-Portable/releases).

**Reply by Telmesomething** [reply]
That thought occurred to me too, but it would mean having to reconfigure everything all over again which I wanted to avoid if possible. But the Portable version is a good idea. I'll have a closer look at that over the weekend and will let you know what happened.

**Comment by Telmesomething** [comment]
Although I haven't tried uninstalling / reinstalling Floorp yet I assume it'll work because I found the reason on the [Mozilla ](https://support.mozilla.org/en-US/questions/1229392)site: regarding the XPCOM screenshot I posted. Therefore I marked your post as Answer.

Thanks anyway for your input Cutter.
